OutOfSchool/OutOfSchool.WebApi/Controllers/V1/ApplicationController.cs

```diff
@@ -167,14 +168,74 @@ public async Task<IActionResult> GetByProviderId(Guid providerId, [FromQuery] Ap
 
         var applications = await applicationService.GetAllByProvider(providerId, filter).ConfigureAwait(false);
 
-        if (applications == null || !applications.Entities.Any())
+        if (CheckApplicationsFor204(applications))
         {
             return NoContent();
         }
 
         return Ok(applications);
     }
 
+    /// <summary>
+    /// Get collection of applications, that have a pending status.
+    /// </summary>
+    /// <param name="providerId">Provider id.</param>
+    /// <returns>List of applications.</returns>
+    /// <response code="200">Entities were found by given Id.</response>
+    /// <response code="204">No entity with given Id was found.</response>
+    /// <response code="500">If any server error occurs.</response>
+    [HasPermission(Permissions.ApplicationRead)]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(SearchResult<ApplicationDto>))]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
```

Comment:
(Comments, OpenApi documentation) List of responses provided in XmlDoc section (`///` comments) is different form list provided in OpenApi section (`ProduceResponseType` attributes). Please align them

Reply:
I dont understand your request too much for this. But, I think, i done what you want me to do.

Reply:
Well? some changes were done, but we need more. The 204 status is not mentioned in `[ProducesResponseType]`. But I see it in xmldoc comment `<response code="204"...>` and in code.

Reply:
Added 204 response. Fixed.